glean-core/src/upload/mod.rs

```diff
@@ -366,7 +394,12 @@ impl PingUploadManager {
     ///
     /// `document_id` - The UUID of the ping in question.
     /// `status` - The HTTP status of the response.
-    pub fn process_ping_upload_response(&self, document_id: &str, status: UploadResult) {
+    pub fn process_ping_upload_response(
+        &self,
+        document_id: &str,
+        status: UploadResult,
+        debug_view_tag: Option<&String>,
```

Comment:
This parameter is undocumented above.
Will this eventually be removed here again, when the debug view tag is also persisted?

Reply:
Yes, it wil be removed. I'll still document it.